tests/jax/test_jax_2d_eldry.py

```diff
@@ -636,8 +637,122 @@ def test_static_eldry(self):
             
             self.assertFalse(grad_failed, 
                          'Incorrect Gradient for static index {}'.format(i))
-        
         pass
 
+        
+    def test_eldry_test_7elem(self):
+        #check force and jacobian
+        rng = np.random.default_rng()

```

Comment:
This needs to pass "seed=____" with some number (e.g., 1023, 42 etc.). Otherwise the test will not always test the exact same case.